integration-tests/src/test/java/oracle/weblogic/kubernetes/ItTwoDomainsLoadBalancers.java

```diff
@@ -363,6 +377,9 @@ public void testDeployAppAndInstallIngressControllers() {
     voyagerHelmParams =
         installAndVerifyVoyager(voyagerNamespace, cloudProvider, enableValidatingWebhook);
 
+    // install and verify Nginx
```

Comment:
Modify the description 
  Test deploy applications and install ingress controllers Traefik. NGINX and Voyager.

Reply:
fixed